7Cav-Alive.Altis/scripts/client/CleanUpLoop.sqf

```diff
@@ -0,0 +1,7 @@
+//base cleanup loop

```

Comment:
If you do this, all clients will be running this script every five minutes. I think this should probably be a server script, not a client one